modules/build-raptor/package.json

```diff
@@ -23,6 +23,7 @@
   },
   "devDependencies": {
     "@types/yargs": "^17.0.7",
-    "@vercel/ncc": "^0.33.1"
+    "@vercel/ncc": "^0.33.1",
+    "axios": "^1.4.0"
```

Comment:
the dependency needs to be added at the package.json of the module that uses it (and, it's not a devDependency).

Reply:
done